test/bdd/fixtures/agent-rest/.env

```diff
@@ -99,3 +99,7 @@ DAVE_ROUTER_WEBHOOK_PORT=10074
 
 
 MYSQL_IMAGE_TAG=8.0.20
+
+COUCHDB_IMAGE=couchdb
+COUCHDB_IMAGE_TAG=2.3.1
```

Comment:
Can we use the latest version ie v3.1 ? https://hub.docker.com/_/couchdb

Reply:
done